SoftLayer/CLI/server/detail.py

```diff
@@ -88,6 +88,12 @@ def cli(env, identifier, passwords, price):
             pass_table.add_row([item['username'], item['password']])
         table.add_row(['users', pass_table])
 
+    if passwords:
```

Comment:
line 91 (and the new line above it) can be removed so that this block is merged into the other if passwords: block